libraries/apollo-api/src/commonTest/kotlin/test/OptionalTest.kt

```diff
@@ -37,7 +37,7 @@ class OptionalTest {
 
   @Test
   fun getOrThrowAbsent() {
-    val optional = Optional.absent<String?>()
+    val optional: Optional<String?> = Optional.absent()
```

Comment:
Inference wasn't working here, I guess that means the `absent()` change is source-breaking in some cases :(

Reply:
Yes it makes more sense like this ✅